src/Stratis.Bitcoin/Controllers/FeatureController.cs

```diff
@@ -19,17 +19,17 @@ public ActionDescription(string description)
 
     public abstract class FeatureController : Controller
     {
-        protected IFullNode FullNode;
+        protected IFullNode FullNode { get; set; }
```

Comment:
Do we want conversion to fields by default? If so: why?

Reply:
This is converted to a property.
Maybe I should convert this back to a field but make them camelCase (I think this is the better way).